### PR TITLE
Slight sysfs reading refactoring

### PR DESCRIPTION
**Pull request by tbzatek**
UDisks 2.10 material. Originally intended for broader use, unused at the end. Still I think it's worth having this, but didn't want to spoil the stable branch.

### CODE REVIEW

File: src/udiskslinuxdevice.c [hunk 1]
```diff
@@ -255,3 +255,95 @@ probe_ata (UDisksLinuxDevice  *device,
...
+guint64
+udisks_linux_device_read_sysfs_attr_as_uint64 (UDisksLinuxDevice  *device,
+                                               const gchar        *attr,
+                                               GError            **error)
+{
+  guint64 ret = 0;
+  gchar *str;
+
+  if ((str = udisks_linux_device_read_sysfs_attr (device, attr, error)))
+    ret = atoll (str);
```
Comment (vojtechtrefny) [inline src/udiskslinuxdevice.c:345]: You should use `g_ascii_strtoull` if we want a 64bit unsigned integer.

**Review by vojtechtrefny — CHANGES_REQUESTED** [review]

File: src/udiskslinuxdevice.c (hunk 1 above)
Comment (tbzatek) [reply src/udiskslinuxdevice.c]: Good point, thanks. There's no `gint` counterpart to `g_ascii_strtoull()`, right?

Comment (vojtechtrefny) [reply src/udiskslinuxdevice.c]: Yes, only `g_ascii_strtoll` and `g_ascii_strtoull`.

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me now, thank you.

**Pull request merged**